include/ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_policy.hpp

```diff
@@ -22,286 +22,128 @@ struct UniversalGemmPipelineAgBgCrPolicy
 
     static constexpr bool TransposeC = true;
 
+    template <typename Problem, typename DataType, index_t kMNPerBlock>
+    CK_TILE_HOST_DEVICE static constexpr auto GetKPack()
+    {
+        constexpr index_t kBlockSize  = Problem::kBlockSize;
+        constexpr index_t kKPerBlock  = Problem::BlockGemmShape::kK;
+        constexpr index_t kMaxVecLoad = 16 / sizeof(DataType);
+        constexpr index_t kMinVecLoad = 4 / sizeof(DataType);
+
+        constexpr index_t total_pixels = kMNPerBlock * kKPerBlock / kBlockSize;
```

Comment:
Pixels is really ambiguous... pixels are usually a vectors of few elements. 
```suggestion
        constexpr index_t elements_per_thread= kMNPerBlock * kKPerBlock / kBlockSize;
```